src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -379,12 +380,25 @@ def build_udaf_apply(
 def build_udtf_apply(
     ast: proto.Expr, udtf_id: int, *args: Tuple[Union[proto.Expr, Any]], **kwargs
 ) -> None:
-    """Builds call to UDTF into ast as Snowpark IR expression."""
+    """Builds call to UDTF into ast as a Snowpark IR expression."""
     expr = with_src_position(ast.apply_expr)
     expr.fn.sp_fn_ref.id.bitfield1 = udtf_id
     build_fn_apply_args(ast, *args, **kwargs)
 
 
+def build_sproc_apply(
+    ast: proto.Expr,
+    sproc_id: int,
+    statement_params: Optional[Dict[str, str]] = None,
+    *args: Tuple[Union[proto.Expr, Any]],
+    **kwargs,
+) -> None:
+    """Builds call to stored procedure into ast as a Snowpark IR expression."""
```

Comment:
nit: Encodes call to stored procedure object.

Reply:
done!